### DISCUSSION TITLE
Decryption of EFS encrypted files

### DISCUSSION THREAD

**Discussion by Winchester678** [sepinf-inc/IPED#1854] [Q&A]
Hello everybody,
I`m currently working with an image that contains files encrypted by Windows EFS. Is there a way to decrypt them with IPED?
Thank you in advance for the help.

**Comment by lfcnassif** [comment]
Unfortunately I think not... Do you know if Sleuthkit (or Autopsy) is able to decrypt them? If yes, we should be able to add the support in the future, since we use Sleuthkit dependency to decode images and file systems.

**Reply by Winchester678** [reply]
No, Autopsy cannot decrypt the files. They are also not labeled as encrypted. 
With IPED, they were also not labeled as encrypted. Nevertheless, thank you very much for the information.

**Reply by lfcnassif** [reply]
Have you checked the "Encrypted Files (entropy)" filter? Does it work for EFS files? If not, maybe we can add detection based on signature, if it exists.

**Reply by Winchester678** [reply]
I tried the filter. Unfortunately, it doesn't mark the files I'm looking for either.

**Reply by Winchester678** [reply]
Sorry, I gave the wrong information. In Autopsy they were not placed in the Encryption Suspected folders.
However, the MFT Entry Header Values said they were EFS encrypted.

**Comment by patrickdalla** [comment]
do you have the master keys? In an AD environment they can be found on AD
server. with these keys the files could be decripted individually, in IPED
without need to change sleuthkit. A new feature should be implemented to do
this,  though

Em sáb., 2 de set. de 2023 05:42, Winchester678 ***@***.***>
escreveu:

**Reply by Winchester678** [reply]
No, I don't have them. But thank you. I can read up on it.
